docs/gaia/gaia.rst

```diff
@@ -501,9 +501,12 @@ To obtain a list of the tables shared to a user type the following:
   tap_schema.columns
   tap_schema.schemas
   tap_schema.key_columns
-  gaiadr1.phot_variable_time_series_gfov
-  gaiadr1.ppmxl_neighbourhood
-  gaiadr1.gsc23_neighbourhood
+  #gaiadr1.phot_variable_time_series_gfov
```

Comment:
this seems to be coming from an unrelated commit, please remove the commit

Reply:
OK, now reading your description it's clear that you added changes for two issues in one PR. 

The general the suggestion is to avoid this, keep PRs self contained around one topic (either multiple issues of a module when part of a bigger refactor/development, or one issue for multiple modules).

Either way, this change doesn't address the issue. The output in this example should be what the commands above give back, and there are no commented out lines in that output.